docs/language/package/pkgtutorial.mdx

```diff
@@ -377,53 +430,32 @@ toit morse_test.toit
 </Tab>
 </Tabs>
 
-## Package.yaml
-
-A package must have a `package.yaml` file at the root of the package.
-
-Since we already added a license, we don't need to worry about the license. However, we need to add the
-"name" and "description" entries.
-
-```yaml
-name: morse_package_tutorial
-description: A tutorial version of the Morse package. Not intended for use as package.
-```
-
-### Dependencies
-
-We don't use dependencies in this package, but for completeness sake here is an example of how we would add one:
-
-```yaml
-dependencies:
-  prefix1:
-    url: github.com/toitware/toit-morse
-    version: ^1.0.0
-  prefix2:
-    url: github.com/toitware/toit-fahrenheit
-    version: ^1.0.0
-```
-
-You normally don't need to worry about them, as they are automatically added when running `jag pkg install <dependency>` or `toit pkg install <dependency>`.
-
 ## Publishing to the Toit package registry
 
-Toit packages are downloaded from the original sources, and (if published publicly) must be accessible through a public Git repository. The URL of the Git repository is part of the ID of a package (together with its version tag). As such, we advise to use the one of the following choices as repository name:
-* "toit-" followed by the package name.
-* the package name, or
+Toit packages are downloaded from the original sources, and (if published publicly) must be
+accessible through a public Git repository. The URL of the Git repository is part of the ID
+of a package (together with its version tag). As such, we advise to use the one of the following
+choices as repository name:
+* "toit-" followed by the package name, or
+* the package name
 
-The easiest way to publish a package is to upload the sources to GitHub and to use the [publish action](https://github.com/toitlang/action-publish).
+The easiest way to publish a package is to upload the sources to GitHub and to use the
+[publish action](https://github.com/toitlang/action-publish).
 
 At each release it automatically takes the version number and informs the Toit package registry about the new
-release. Just make sure that the version is of the form `v1.0.0` (or similar). Version numbers follow [semantic versioning](https://semver.org) and tags must be valid semver versions with a prefixed 'v'.
+release. Just make sure that the version is of the form `v1.0.0` (or similar). Version numbers follow
```

Comment:
What does it mean to be of the form x or similar? I think I'd drop "(or similar)".